server/auth/types/multiple/multi_auth.ts

```diff
@@ -17,11 +17,11 @@ import {
   SessionStorageFactory,
   IRouter,
   ILegacyClusterClient,
-  OpenSearchDashboardsRequest,
   Logger,
   LifecycleResponseFactory,
+  OpenSearchDashboardsRequest,
   AuthToolkit,
-} from '../../../../opensearch-dashboards/server';
+} from 'opensearch-dashboards/server';
```

Comment:
Not sure why this changed, but I was getting errors on my local, and it seems other files are importing it like this, so I changed it to be consistent.